main.go

```diff
@@ -139,6 +140,8 @@ func main() {
 		"The duration of time that artifacts will be kept in storage before being garbage collected.")
 	flag.IntVar(&artifactRetentionRecords, "artifact-retention-records", 2,
 		"The maximum number of artifacts to be kept in storage after a garbage collection.")
+	flag.BoolVar(&recoverPanic, "RecoverPanic", true,
+		"Indicates whether the panic caused by reconcile should be recovered.")
```

Comment:
I don't think we need to make this configurable. As far as I know, we don't have a use case to allow the controller to panic. Recover panic should always be enabled everywhere.

Reply:
Thankyou for the review and I agree to you point. I’ll make the changes accordingly.